justfile

```diff
@@ -90,7 +96,7 @@ bin-kind:
 # Clean
 
 # Clean kind test env
-clean-test-kind:
+test-kind-clean:
```

Comment:
I think this should be `clean-test-kind` as this is not a test, but a cleanup script instead.